recipe/meta.yaml

```
@@ -105,3 +94,4 @@ about:
 extra:
   recipe-maintainers:
     - jakirkham
+    - h-vetinari
```

Feel free to add me if this going to be an arrow dependency.
```suggestion
    - h-vetinari
    - xhochy
```